content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,478 @@
+# How to build a task manager with angular.
+
+## Introduction
+As developers, it is common practice to build simple applications. This will help solidify ones knowledge about a particular programming language or framework. 
+
+In this article, we will build a simple task manager application (AKA To-do app) using  [angular](https://angular.io/docs). Angular is a JavaScript framework.
+
+
+## What is a Task Manager?
+A task manager writes, organizes, and rearranges tasks more efficiently. The task manager we will build will allow the user to add a task, mark the task as complete, and delete the task.
+
+
+## Key Takeaways From This Article
+1. How to create and structure an angular project.
+2. How to create modules and components in angular.
+3. Build a simple application.
+4. Save tasks to local storage
+
+
+## Prerequisites
+
+Before we get started, here are a few prerequisites for developing a task manager via Angular;
+
+- You should have [Node.js](https://nodejs.org/en/) installed on your computer.
+- You should be fairly familiar with [Angular](https://angular.io/docs) and JavaScript. Knowledge of [TypeScript](https://www.typescriptlang.org/) is useful but not requisite.
+
+
+## Overview
+1. Create a new angular project
+2. Create a component
+3. Add task
+4. Mark task as done
+5. Delete a task
+6. Save to local storage
+
+
```

Comment:
This overview should appear immediately after introduction. Use the correct numbering for this part.